resources/views/dashboard.blade.php

```diff
@@ -176,24 +178,28 @@ class="btn btn-sm btn-outline-primary">Assistants & Shares</a>
             </div>
         @endif
         <div class="col-md-6 mx-auto">
-            @if (Carbon\Carbon::parse(config('con.reg_end_date'))->isFuture())
+            @if (Carbon\Carbon::parse(config('con.reg_end_date'))->isFuture() ||
+                    Carbon\Carbon::parse(config('con.assistant_end_date'))->isFuture())
                 <div class="card mb-2">
-                    <div class="card-body text-center">
-                        <h5 class="card-title display-6">Join an existing Dealership</h5>
-                        <p class="card-text lead">You have been invited by an existing dealership to share their space at
-                            the Dealers' Den.</p>
-                        <a href="{{ route('join') }}" class="btn btn-lg btn-outline-primary">I have an invitation code</a>
-                    </div>
-                </div>
-            @endif
-            @if (Carbon\Carbon::parse(config('con.assistant_end_date'))->isFuture())
-                <div class="card mb-2">
-                    <div class="card-body text-center">
-                        <h5 class="card-title display-6">Become a Dealer Assistant</h5>
-                        <p class="card-text lead">You have been invited to support an existing Dealership as a Dealer Assistant.
-                        </p>
-                        <a href="{{ route('join') }}" class="btn btn-lg btn-outline-primary">I have an invitation code</a>
-                    </div>
+                    <form action="{{ route('invitation.join') }}" method="post">
+                        <div class="card-body text-center">
+                            <h5 class="card-title display-6">Join an existing Dealership</h5>
+                            <p class="card-text lead">You have been invited by an existing dealership to share their space
+                                or assist them at the Dealers’ Den?<br>
+                                Then enter the invite code they provided you with below!
```

Comment:
Bit confusing, suggestion:
"Then enter below the invite code they provided you with"